apps/docs/content/components/RiverAccordion.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: River Accordion
+description: Use the river accordion to create collapsible content panels with associated visuals.
+source: https://github.com/primer/brand/blob/main/packages/react/src/river/RiverAccordion/RiverAccordion.tsx
+figma: 'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=16649-1497&p=f&t=lzotdAbtGDWpDEej-11'
+storybook: '/brand/storybook/?path=/story/components-riveraccordion--playground'
```

Comment:
Could we create a default example and link to that instead of playground please?